open-company/open-company-web
# Fix ellipses menu for viewer users

Card: https://trello.com/c/pxVcxj3V

To test:
- go to AP as admin or contrib
- [x] do you see a horizontal ellipses and a share button when you hover a post? Good
- [x] do you see a menu with at least one voice when you click the ellipse? Good
- invite a viewer
- accept the invitation
- check a post in AP or in a single section
- [x] do you see the share button? Good
- [x] is it right aligned with the rest of the card content? Good
- [x] do you NOT see an ellipses beside it? Good
